examples/ffi/c_calling_rust/BUILD

```diff
@@ -0,0 +1,20 @@
+load("@//rust:rust.bzl", "rust_library", "rust_test", "rust_binary")
```

Comment:
Nit: Other examples have a visibility defined:
```
package(default_visibility = ["//visibility:public"])
```

Reply:
Should we weaken the rest of the examples' visibilities?

Reply:
I'm not excited about anyone depending on this or other examples